### DISCUSSION TITLE
Undefined class 'RustLibWasmModule'.

### DISCUSSION THREAD

**Discussion by lunxinfeng** [fzyzcjy/flutter_rust_bridge#2400] [General]
I integrated frb through `flutter_rust_bridge_codegen integrate`,  Idea prompts an error, which does not affect compilation. However, an error is prompted when using code gen (like build_runner). Is it because of missing configuration?

![frb_web](https://github.com/user-attachments/assets/97958c44-6263-47d6-bf3c-f813dce50952)

It seems that creating a new program through `flutter_rust_bridge_codegen create my_app` is normal.

**Comment by fzyzcjy** [comment]
First guess: in your pubspec.yaml, do you specify a high enough Dart/Flutter version? because I see "type" is red

**Reply by lunxinfeng** [reply]
thks, you are right, I fix it by this:

```yaml
#environment:
#  sdk: ">=3.0.0 <4.0.0"
environment:
  sdk: ^3.5.3
```

**Reply by fzyzcjy** [reply]
You are welcome!
